OracleWebLogic/samples/wls-k8s-domain/deploy.sh

```diff
@@ -0,0 +1,5 @@
+kubectl create -f  k8s/secrets.yml 
```

Comment:
This script needs a proper shebang line: `#!/bin/bash`

However, the value of the script is questionable.